webapp/src/components/layout/Layout.tsx

```diff
@@ -11,18 +11,19 @@ type Props = {
 export default function Layout({ children }: Props) {
   const [scrollOk, setScrollOk] = useState<boolean>(false);
 
-  const boxRef = useRef<HTMLDivElement>(null);
   const scrollOff: number = document.querySelectorAll('.row')[0]?.getBoundingClientRect().top;
+
   function handleScroll() {
     if (document.querySelectorAll('.row')[0].getBoundingClientRect().top >= scrollOff) setScrollOk(false);
     else setScrollOk(true);
```

Comment:
Ça fait moins mal aux yeux et c'est plus simple à lire comme ça:

```suggestion
    if (document.querySelectorAll('.row')[0].getBoundingClientRect().top >= scrollOff) {
        setScrollOk(false);
    } else {
        setScrollOk(true);
    }
```